source/tests/common/dpmodel/test_repformer_graph_ops.py

```
@@ -0,0 +1,469 @@
+        torch.from_numpy(g1),
+        torch.from_numpy(g2_flat),
+        torch.from_numpy(h2_flat),
+        torch.from_numpy(src),
+        torch.from_numpy(dst),
+        torch.from_numpy(mask_flat),
+        torch.from_numpy(sw_flat),
+        n_total,
+        NNEI,
+        t_pairs,
+    )
+    for r, g in zip(ref, got):
```

Ruff (0.15.20)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)